contributing.md

```diff
@@ -30,3 +30,13 @@ Guidelines](https://opensource.google/conduct/).
 All submissions, including submissions by project members, require review. We 
 use [GitHub pull requests](https://docs.github.com/articles/about-pull-requests)
 for this purpose.
+
+After creating a PR, a contributor needs to add `/gcbrun` to the comment to trigger CI
```

Comment:
can anyone leave the comment to trigger CI or does it need to be a maintainer? "contributor" sounds like the PR author.

Reply:
The action itself says "Needs /gcbrun from a collaborator". So it's not everyone. I updated the word here to "collaborator".